### PR TITLE
Added more functions in brick ops

### PR DESCRIPTION
**Pull request by aujjwal-redhat**
Added two functions in brick ops:
* delete_bricks
* is_broken_symlinks_present_on_bricks

Fixes: #678

Signed-off-by: Ayush Ujjwal <aujjwal@redhat.com>

### CODE REVIEW

File: common/ops/gluster_ops/brick_ops.py [hunk 1]
```diff
@@ -766,3 +766,58 @@ def get_brick_processes_count(self, node: str) -> int:
...
+                                                brick_node,
+                                                False)
+            if ret['error_code'] == 0:
+                self.logger.error(f"Unable to delete brick {brick_path}"
+                                  f" on node {brick_node}")
+                _rc = False
+        return _rc
+
+    def is_broken_symlinks_present_on_bricks(self, node: str,
+                                             volname: str):
```
Comment (nik-redhat) [inline common/ops/gluster_ops/brick_ops.py:796]: Mention the return type

File: common/ops/gluster_ops/brick_ops.py [hunk 2]
```diff
@@ -766,3 +766,58 @@ def get_brick_processes_count(self, node: str) -> int:
...
+        """
+        Checks if the backend bricks have broken symlinks.
+
+        Args:
+            node (str): Node on which command has to be executed.
+            volname (str): Name of the volume
+
+        Returns: Bool
```
Comment (nik-redhat) [inline common/ops/gluster_ops/brick_ops.py:804]: Break to next line

File: common/ops/gluster_ops/brick_ops.py [hunk 3]
```diff
@@ -766,3 +766,58 @@ def get_brick_processes_count(self, node: str) -> int:
             return count_of_proc
         else:
             return None
+
+    def delete_bricks(self, bricks_list: list):
```
Comment (nik-redhat) [inline common/ops/gluster_ops/brick_ops.py:770]: mention the return type

**Review by nik-redhat — CHANGES_REQUESTED** [review]

File: common/ops/gluster_ops/brick_ops.py (hunk 1 above)
Comment (aujjwal-redhat) [reply common/ops/gluster_ops/brick_ops.py:796]: Ack

File: common/ops/gluster_ops/brick_ops.py (hunk 2 above)
Comment (aujjwal-redhat) [reply common/ops/gluster_ops/brick_ops.py:804]: Ack

File: common/ops/gluster_ops/brick_ops.py (hunk 3 above)
Comment (aujjwal-redhat) [reply common/ops/gluster_ops/brick_ops.py:770]: Ack

**Review by nik-redhat — APPROVED** [review]

**Pull request merged**